app/services/app_store_client.rb

```diff
@@ -40,6 +40,17 @@ def create_subscription(payload)
     end
   end
 
+  def create_event(submission_id, payload)
```

Comment:
Nit: If the AppStoreClient has a `create_event` method, that suggests that method should only be usable to create an event. But you could in theory call `create_event` with an arbitrary length array of events. I reckon either this should be called `create_event`, or it should take a _single_ event object as its second argument and create a payload from that.

Reply:
I think I like change the name to `create_events` to avoid this method needing to know about payload creation...